Dockerfile

```diff
@@ -25,10 +28,11 @@ RUN virtualization/Docker/setup_docker_prereqs
 
 # Install hass component dependencies
 COPY requirements_all.txt requirements_all.txt
+
 # Uninstall enum34 because some dependencies install it but breaks Python 3.4+.
 # See PR #8103 for more info.
 RUN pip3 install --no-cache-dir -r requirements_all.txt && \
-    pip3 install --no-cache-dir mysqlclient psycopg2 uvloop cchardet cython
+    pip3 install --no-cache-dir mysqlclient psycopg2 uvloop cchardet cython opencv-python-headless
```

Comment:
I'm always a bit wary if we need to install things for every single user of Home Assistant. Why can't we install this on demand?

Reply:
It looks like it will also work without opencv. We should remove this.

Reply:
The thought here was to make it as easy as possible for those using the official docker image.  OpenCV is pre-built in the Hass.io images [here](https://github.com/home-assistant/hassio-homeassistant/blob/master/base/Dockerfile#L36-L57), though from source.  The pip package makes it easier to install.

I was aiming for consistency.